pandas/tseries/offsets.py

```diff
@@ -2947,3 +2947,39 @@ def generate_range(start=None, end=None, periods=None,
 ])
 
 prefix_mapping['N'] = Nano
+
+# Convenience objects
+bday = BDay()
+businessDay = bday
+day = DateOffset()
+
+cday = CDay()
+customBusinessMonthEnd = CBMonthEnd()
+customBusinessDay = CustomBusinessDay()
```

Comment:
you wholesale added these back I never said that
I just want them deprecated 
I don't say this was easy